charts/dvpe-certificate-issuer-controller/templates/_helpers.tpl

```diff
@@ -1,3 +1,7 @@
 {{- define "image.url" -}}
 {{- printf "\"%s/%s:%s\"" .Values.issuercontroller.spec.image.repository .Values.issuercontroller.spec.image.name .Values.issuercontroller.spec.image.tag -}}
-{{- end -}}
\ No newline at end of file
+{{- end -}}
+
+{{- define "container.name" -}}
+{{- default .Release.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
```

Comment:
if we don't need the `issuercontroller.spec.image.name` value anymore, we should delete it from values yaml and the readme, or am I wrong and we use it still somewhere else?

Reply:
Hi @herrLierb ,

the issuercontroller.spec.image.name does not have to be the name of the container name.
The separation of container name and image name is necessary, because the container name does not support special chars like "/". But the container image is often stored with subpaths (Example: container-image/feature/xyz).

I hope it make clear why this patch is required.

Thanks,
Best,
C.